src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -58,10 +60,25 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                     this.WriteLine("LogRecord.StateValues (Key:Value):");
                     for (int i = 0; i < logRecord.StateValues.Count; i++)
                     {
-                        this.WriteLine($"{logRecord.StateValues[i].Key,-RightPaddingLength}{logRecord.StateValues[i].Value}");
+                        // Special casing {OriginalFormat}
+                        // See https://github.com/open-telemetry/opentelemetry-dotnet/pull/3182
+                        // for explanation.
+                        if (logRecord.StateValues[i].Key.Equals("{OriginalFormat}") && string.IsNullOrEmpty(body))
+                        {
+                            body = logRecord.StateValues[i].Value as string;
+                        }
+                        else
+                        {
+                            this.WriteLine($"{logRecord.StateValues[i].Key,-RightPaddingLength}{logRecord.StateValues[i].Value}");
+                        }
                     }
                 }
 
+                if (!string.IsNullOrEmpty(body))
+                {
+                    this.WriteLine($"{"Body:",-RightPaddingLength}{body}");
```

Comment:
This might be confusing since `Body` is not a first-class citizen in `LogRecord`.

Reply:
Maybe consider something like:

```
"{OriginalFormat}" (a.k.a. Body): "Hello from {name} {price}."
```

Reply:
Upon reading again, yes, I agree.